src/k8s/pkg/k8sd/api/cluster_remove.go

```diff
@@ -32,29 +32,32 @@ func (e *Endpoints) postClusterRemove(s *state.State, r *http.Request) response.
 		defer cancel()
 	}
 
+	log := log.FromContext(s.Context).WithValues("name", req.Name)
```

Comment:
nit: `WithName`?

Reply:
note: we are adding the node name as `name=foo`. `log.WithName("test")` would give us a field with `logger="k8sd.test"` which is not what we want here